### PR TITLE
wicked: Add ifreload-3.sh -- if reload with bond bridges

### PR DESCRIPTION
**Pull request by cfconrad**
Ticket: https://progress.opensuse.org/issues/66949
issue:  https://gitlab.suse.de/wicked-maintainers/wicked/-/issues/156
bug:    http://bugzilla.suse.com/show_bug.cgi?id=1168155

Wicked developers provide testscript for ifreload bug when using bridges
+ bond interface with certain VLAN configs.

- Verification run:
  - http://cfconrad-vm.qa.suse.de/tests/7691 (ok)
  - http://cfconrad-vm.qa.suse.de/tests/7688 (failed)

### CODE REVIEW

**Comment by cfconrad** [comment]
Set to WIP because of http://cfconrad-vm.qa.suse.de/tests/7637#step/t13_bridge_ifreload_all/7
any hint?

**Comment by asmorodskyi** [comment]
I think it is because you messing up with calling backed functions . not sure if it is expected at all from test code ...

File: tests/wicked/startandstop/sut/t13_bridge_ifreload_all.pm [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    $args{result} //= 'failed';
+    $title =~ s/:/_/g;
+    my $filename     = $self->next_resultname('txt', $title);
+    my $extra_result = {
+        _source => 'parser',
+        result  => $args{result},
+        text    => $filename,
+        title   => $title,
+    };
+    $self->write_resultfile($filename, $content);
```
Comment (asmorodskyi) [inline tests/wicked/startandstop/sut/t13_bridge_ifreload_all.pm:39]: @okurz  , @Martchus  - WDYT ?

Comment (cfconrad) [reply tests/wicked/startandstop/sut/t13_bridge_ifreload_all.pm]: I think it is more a question of introducing something like 
```
$self->record_console_test_result('ifreload-3', $output, result => $result);
```
I think this is really useful for console based tests as it makes the output in openqa-ui looking quite nice.

I'm going to create a PR in os-autoinst, but let's not stop this PR with such look&feel thing, or?

**Comment by asmorodskyi** [comment]
only Travis blocking me from merging

**Pull request merged**